blocks/two-up-post/style.scss

```diff
@@ -21,7 +21,7 @@
   div.wp-block-wp-newsletter-builder-post {
     max-width: 300px;
     padding: 0 10px;
-    width: 50%;
+    flex: 1 1 50%;
```

Comment:
this probably won't work in the email client.

Reply:
this may not show up there either... if so, ignore me.

Reply:
Email styling is literally the worst

Reply:
It seems to show up, so I reverted back to the`width` property